pandas/tests/generic/test_finalize.py

```diff
@@ -251,7 +251,6 @@
             operator.methodcaller("isin", pd.DataFrame({"A": [1]})),
         ),
     ),
-    (pd.DataFrame, frame_data, operator.methodcaller("swapaxes", 0, 1)),
```

Comment:
It seems that this file is testing all existing finalize methods, so I removed `swapaxes` due to its depreciation.